frontend/components/Token.js

```diff
@@ -0,0 +1,9 @@
+import * as SecureStore from 'expo-secure-store';
+
+export default async function Token(key='token') {
```

Comment:
change name to setToken.
Take the parameter token here like
setToken(token)
The token parameter should be the value returned from the api